browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2524,6 +2541,16 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		builder._preventDocumentLosingFocusOnClick(control.container);
 	},
 
+	_handlePressAndHold: function(data) {
```

Comment:
can we put this inside `_mapDispatchToolItem` ? 
we are moving with widgets to be real classes/objects, I fear someone will forgot this when moving from this file.

`var handlePressAndHold = function(data) { ... };`

Reply:
sure, got it!